### PR TITLE
Issue #8 - Personal page

### PR DESCRIPTION
**Pull request by perstarkse**
## Description

Added my personal page: 
image

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

Relates to [#8].

Your ENS/address: perstark.eth

### CODE REVIEW

**Review by damianmarti — CHANGES_REQUESTED** [review]
This is looking great!

Please fix the lint issue:

./app/builders/0xADAa82d99Ca31379EC8417515A75A7ce3A4b4044/page.tsx
7:7  Warning: Using `` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element

File: packages/nextjs/app/builders/0xADAa82d99Ca31379EC8417515A75A7ce3A4b4044/page.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        className="w-64 h-80 object-cover rounded-md mx-auto mt-1 sm:mx-0"
+      />
+      <div className="flex flex-col gap-0 max-w-sm">
+        <h1 className="text-3xl">Per Stark</h1>
+        <Address size="base" address="0xADAa82d99Ca31379EC8417515A75A7ce3A4b4044" />
+        <p className="text-sm italic">Software developer / Psychologist</p>
+        <hr className="border-accent" />
+        <p className="text-sm mt-2">
+          I am a full stack developer with a previous career as a clinical psychologist. Been in crypto space on and off
+          since 2017. Happy to have connected with BuidlGuidl. <br /> <br /> I also work as a part time politician in my
```
Comment (damianmarti) [inline packages/nextjs/app/builders/0xADAa82d99Ca31379EC8417515A75A7ce3A4b4044/page.tsx:20]: You can use another **p** element to split this text, and avoid using the **br** elements.

**Review by damianmarti — APPROVED** [review]
Great job!

And really interesting career background :-)

**Comment by perstarkse** [comment]
> Great job!
> 
> And really interesting career background :-)

Thanks 🙏

Truly having very fun coding. Really enjoying the process of open source developing as well. PR processes like these provide ample learning!

**Pull request merged**